### DISCUSSION TITLE
Question - Extendable Weather Skins

### DISCUSSION THREAD

**Discussion by RightClickClyde** [mpurses/Sonder#41] [Q&A]
Is it possible to get the drop down portion of the Extended Weather Skins to permanently stay exposed? Is it as simple as editing it? I like a clean look where the borders of all my skins are in a subjectively perfect spot and when its retracted it leaves an awkward space below it. Example with Weather Skin extended (by hovering):
![image](https://user-images.githubusercontent.com/48803539/161935068-b9025733-a407-4358-9c65-ce087e776340.png)

**Comment by mpurses** [comment]
Right click > Edit skin.  Modify these lines:
![image](https://user-images.githubusercontent.com/642218/162107333-0a841cbb-6680-48db-a92f-7f07f739d065.png)

I'll probably make this a checkbox in a future version.
